### DISCUSSION TITLE
Edge 商店搜索出现该扩展的山寨品，请大家一起举报。

### DISCUSSION THREAD

**Discussion by Alphabet1226** [immersive-translate/immersive-translate#366] [General]
### 概述 | Summary

在微软Edge商店搜索“沉浸式翻译”，出来的第一个结果是山寨的。该山寨扩展甚至连Logo、名称、宣传图都复制了过去，一模一样，但作者显示是“三商电脑软件开发有限公司”。

这种山寨扩展99%都是带病毒的，在历史上出现过的例子数不胜数——把开源扩展偷过去，加上恶意代码，再改个名字发布来赚钱。请大家一起举报（这里是[直达链接](https://microsoftedge.microsoft.com/addons/report/bnhjlfflcalkgdpphmffbcmplfomgbek?hl=zh)），很简单，15秒钟就能完成。

![image](https://user-images.githubusercontent.com/51783873/232354919-86e06224-6a47-43b4-a975-f931161c4b25.png)

**Comment by Alphabet1226** [comment]
但看起来discussions通常是不会有人看的

**Comment by idjj** [comment]
已经举报

Alphabet1226 ***@***.***>于2023年4月18日 周二20:37写道：

**Comment by tcmtom** [comment]
好像已经看不到了
